File: frontend/src/app/models/product-model.ts
import StoreModel from './store-model';

export default interface ProductModel {
  productId: number;
  name: string;
  description: string;
  price: number;
  manufacturerId: number;
  manufacturerName: string;
  categoryId: number;
  categoryName: string;
  store: StoreModel | null;
  imageUrls: string[];
}

File: services/OrderService/OrderService.Application/Src/UseCases/ProductDescription/ProductModel.cs
namespace OrderService.Application.UseCases.ProductDescription;

public class ProductModel
{
    public int ProductId { get; set; }
    public string Name { get; set; } = string.Empty;
    public string Description { get; set; } = string.Empty;
    public decimal Price { get; set; }
    public int ManufacturerId { get; set; }
    public int CategoryId { get; set; }
}
